src/components/PostDetails.tsx

```diff
@@ -157,9 +98,7 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
           </button>
         )}
 
-        {loaded && !hasError && visible && (
-          <NewCommentForm onSubmit={addComment} />
-        )}
+        {!loading && !error && visible && <NewCommentForm />}
```

Comment:
The NewCommentForm component is rendered based on the 'visible' state. Ensure that this component is correctly handling the submission of new comments and updating the comments list accordingly.